src/back/glsl/mod.rs

```diff
@@ -2960,6 +2961,78 @@ impl<'a, W: Write> Writer<'a, W> {
                     Mf::Transpose => "transpose",
                     Mf::Determinant => "determinant",
                     // bits
+                    Mf::CountTrailingZeros => {
+                        if self.options.version.supports_integer_functions() {
+                            match *ctx.info[arg].ty.inner_with(&self.module.types) {
+                                crate::TypeInner::Vector { size, kind, .. } => {
+                                    let s = back::vector_size_str(size);
+
+                                    if let crate::ScalarKind::Uint = kind {
+                                        write!(self.out, "uvec{s}(findLSB(")?;
+                                        self.write_expr(arg, ctx)?;
+                                        write!(self.out, ") - ivec{s}(1))")?;
+                                    } else {
+                                        write!(self.out, "mix(findLSB(")?;
+                                        self.write_expr(arg, ctx)?;
+                                        write!(self.out, ") - ivec{s}(1), ivec{s}(0), lessThan(")?;
```

Comment:
`findLSB` doesn't return `0` when the negative bit is set. So these `lessThan` checks are no longer needed.

Reply:
True - We can use directly lsb